### DISCUSSION TITLE
Strange Error message !!

### DISCUSSION THREAD

**Discussion by jeffme1967** [fedorovvl/tso_client#153] [Q&A]
Hi and thanks for your great job !

I've a big issue whith the TSO client :
I've downloaded the new version but it doesn't work...
My OS is Windows 7 64 bits

here is a screenshot of the strange message ^^
![Client exe ERROR](https://user-images.githubusercontent.com/119007505/203850343-54426ba3-7c87-423e-b79f-9229e03a6870.jpg)

Thanks for your help !

regards
Jeffme1967

**Comment by Jazzyguy67** [comment]
Hi, did your antivirus stop the download?

**Reply by jeffme1967** [reply]
Hi and thanks !
I'm sure it didn't because I created an exception for the client.exe.
I'll reinstall my OS and try again ^^
